src/app/rpmostree-builtin-status.c

```diff
@@ -267,10 +293,45 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
           print_kv ("Timestamp", max_key_len, timestamp_string);
         }
 
+      if (!g_variant_dict_lookup (dict, "live-inprogress", "&s", &live_inprogress))
+        live_inprogress = NULL;
+      if (!g_variant_dict_lookup (dict, "live-replaced", "&s", &live_replaced))
+        live_replaced = NULL;
+      const gboolean have_live_changes = live_inprogress || live_replaced;
+
       if (is_locally_assembled)
-        print_kv ("BaseCommit", max_key_len, base_checksum);
-      if (opt_verbose || !is_locally_assembled)
-        print_kv ("Commit", max_key_len, checksum);
+        {
+          if (have_live_changes)
+            print_kv ("BootedBaseCommit", max_key_len, base_checksum);
+          else
+            print_kv ("BaseCommit", max_key_len, base_checksum);
+          if (opt_verbose || have_any_live_overlay)
```

Comment:
Yeah.   I agree that'd be more accurate, but I think it'd be better to add the notion of "deployment referencing" to the core?